packages/app/src/systems/Core/components/Mnemonic/Mnemonic.tsx

```diff
@@ -49,11 +49,6 @@ export function Mnemonic({
     fillArray(initialValue, format)
   );
 
-  async function handleCopy() {
```

Comment:
Should we still allow a user to copy their secret phrase or are we basically forcing them to write it down physically somewhere?